homeassistant/components/sia/translations/en.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file is automatically generated. Please do not commit.

Reply:
Done